src/System.Dynamic.Runtime/tests/Dynamic.Simple/DelegateTest.cs

```diff
@@ -24,5 +24,38 @@ public static void DelegateTest_RunTest()
             int result = d(4, 6);
             Assert.Equal(24, result);
         }
+
+        private static int Foo(string s, dynamic d, object o)
+        {
+            return s?.Length + d?.ToString()?.Length + o?.ToString()?.Length;
+        }
```

Comment:
I'm confused about what is this trying to accomplish. If any of the parameters is `null`, you'll get a `RuntimeBindingException` about not being able to convert `null` to `int`. I get that the parameters are never `null`, so it doesn't really matter, but in that case, what is the point of using `?.` in the first place?